docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -251,6 +252,9 @@ Column name             Type                            Description
 For tables without a table definition file, the ``_key_corrupt`` and
 ``_message_corrupt`` columns will always be ``false``.
 
+Note, internal column prefix is configurable by ``kafka.internal-field-prefix``
+configuration property and defaults to ``_``
```

Comment:
Probably move this before the table with something like:

```rst
Note that the internal column name prefix is configurable by setting
``kafka.internal-field-prefix`` catalog configuration property and it defaults
to ``_``. In case you have configured a different value the internal column
names will appear different from the table below.
```

(Users often skim things and don't notice impact of some config unless  you spell it out clearly).